scalafix-testkit/src/main/scala/scalafix/testkit/SyntacticRuleSuite.scala

```diff
@@ -8,16 +8,20 @@ import scalafix.syntax._
 
 import org.scalatest.FunSuiteLike
 
-class SyntacticRuleSuite(rule: Rule) extends FunSuiteLike with DiffAssertions {
```

Comment:
The migration manager (mima) is complaining because of this change, scalafix-testkit is a published artifact to help scalafix users develop custom rules in repositories outside of scalacenter/scalafix.

I think we can make this change without breaking compatibility by changing this parameter to `defaultRule: Rule = Rule.empty`.

Reply:
Done.